crates/re_types/definitions/rerun/components/clear_is_recursive.fbs

```diff
@@ -14,7 +14,7 @@ struct ClearIsRecursive (
   "attr.arrow.transparent",
   "attr.python.aliases": "bool",
   "attr.python.array_aliases": "bool, npt.NDArray[np.bool_]",
-  "attr.rust.derive": "Copy, PartialEq, Eq",
+  "attr.rust.derive": "Copy, PartialEq, Eq, Default",
```

Comment:
Here it would also be nice to be explicit and explain why `recursive=false` should be the default